cloudslang-entities/src/main/java/io/cloudslang/lang/entities/bindings/values/SensitiveValue.java

```diff
@@ -68,7 +79,12 @@ public void encrypt() {
     protected String encrypt(Serializable originalContent) {
         byte[] serialized = serialize(originalContent);
         String serializedAsString = Base64.encodeBase64String(serialized);
-        return EncryptionProvider.get().encrypt(serializedAsString.toCharArray());
+        Encryption encryption = EncryptionProvider.get();
```

Comment:
we don't need to do that as we use obfuscation only in this particular case and the decrypt method handles both deobfuscation and decryption. @lucian-cm WDYT?